apps/api/src/app/vercel-integration/usecases/setup-integration/setup-integration.command.ts

```diff
@@ -0,0 +1,7 @@
+import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
+import { IsDefined } from 'class-validator';
+
+export class SetupIntegrationCommand extends EnvironmentWithUserCommand {
+  @IsDefined()
+  code: string;
```

Comment:
Maybe we can use a more precise name here instead of just `code`? For me reading this the first time I don't really know what token / specific part of vercel is meant here.